app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Toggle.kt

```diff
@@ -71,35 +76,9 @@ fun HedvigToggle(
   modifier: Modifier = Modifier,
   toggleStyle: ToggleStyle = ToggleDefaults.toggleStyle,
 ) {
-  val initialContainerColor = toggleColors.containerColor(false)
-  val pulsatingContainerColor = toggleColors.containerColor(true)
-  val containerColor = remember { Animatable(initialContainerColor) }
-  val initialLabelColor = toggleColors.labelColor(false)
-  val pulsatingLabelColor = toggleColors.labelColor(true)
-  val labelColor = remember { Animatable(initialLabelColor) }
-  val initialDescriptionColor = toggleColors.descriptionColor(false)
-  val pulsatingDescriptionColor = toggleColors.descriptionColor(true)
-  val descriptionColor = remember { Animatable(initialDescriptionColor) }
-  LaunchedEffect(turnedOn) {
-    if (turnedOn) {
-      launch {
-        containerColor.animateTo(pulsatingContainerColor, animationSpec = animationSpec)
-        containerColor.animateTo(initialContainerColor, animationSpec = animationSpecExit)
-      }
-      launch {
-        labelColor.animateTo(pulsatingLabelColor, animationSpec = animationSpec)
-        labelColor.animateTo(initialLabelColor, animationSpec = animationSpecExit)
-      }
-      launch {
-        descriptionColor.animateTo(pulsatingDescriptionColor, animationSpec = animationSpec)
-        descriptionColor.animateTo(initialDescriptionColor, animationSpec = animationSpecExit)
-      }
-    } else {
-      launch { containerColor.animateTo(initialContainerColor, animationSpec = animationSpec) }
-      launch { labelColor.animateTo(initialLabelColor, animationSpec = animationSpec) }
-      launch { descriptionColor.animateTo(initialDescriptionColor, animationSpec = animationSpec) }
-    }
-  }
+  val containerColor = toggleColors.containerColor(turnedOn)
+  val labelColor = toggleColors.labelColor(turnedOn)
+  val descriptionColor = toggleColors.descriptionColor(turnedOn)
```

Comment:
Yup this is looking great, I feel like the color animation being an impl detail inside the `labelColor` etc is a nice thing. That way if it's animated or not does not matter really here.

We could also use the `by` syntax here too, to avoid the `.value` call further down, if possible.